src/_includes/assets/js/app.js

```diff
@@ -7,11 +7,7 @@ var navToggle = document.getElementById('navigation-toggle')
 var nav = document.getElementById('navigation')
 
 navToggle.addEventListener('click', function() {
-  if (nav.classList.contains('open')) {
-    nav.classList.remove('open')
-  } else {
-    nav.classList.add('open')
-  }
+  nav.classList.toggle('open')
```

Comment:
what's the difference ? https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods

Reply:
The difference (I requested the change) is that there's no reason to write your own version of toggle, with two separate `classlist` method calls when there is a single `classlist` method that does the exact thing.  The latter is more clear (it does what it says, toggle classes,) concise (one line) and faster (one call which could be optimized versus two cobbled together)